src/App.js

```diff
@@ -1,21 +1,103 @@
-import React, { useState, useEffect } from 'react';
-import PokemonList from './PokemonList';
-import axios from "axios";
-import './App.css';
+import React, { useState, useEffect } from "react";
+import PokemonList from "./PokemonList";
+import Axios from "axios";
+import "./App.css";
+import PaginationFunction from "./Pagination";
+import Spinner from 'react-bootstrap/Spinner'
+import Limit from "./selectLimit";
+import pokeBall from './Images/pokeBall.png'
 
 function App() {
-  const [pokemon, setPokemon] = useState([]);
+  const [pageLimit, setPageLimit] = useState(20);
+  const [currentPage, setcurrentPage] = useState(1);
+  const offset = (currentPage - 1) * pageLimit;
+  const url = `${process.env.REACT_APP_POKE}?offset=${offset}&limit=${pageLimit}`;
```

Comment:
This should be moved inside `fetchPokemonList` and that function should be using offset and pageLimit as params